lib/src/client.dart

```diff
@@ -57,11 +57,37 @@ abstract class MeiliSearchClient {
   Future<Map<String, String>> getDumpStatus(String uid);
 
   /// Get the public and private keys.
-  Future<Map<String, String>> getKeys();
+  Future<List<Key>> getKeys();
+
+  /// Get a specific key by key.
+  Future<Key> getKey(String key);
+
+  /// Create a new key.
+  Future<Key> createKey(
+      {DateTime? expiresAt,
+      String? description,
+      required List<String> indexes,
+      required List<String> actions});
+
+  /// Update a key.
+  Future<Key> updateKey(String key,
+      {DateTime? expiresAt,
+      String? description,
+      List<String>? indexes,
+      List<String>? actions});
+
+  /// Delete a key
+  Future<bool> deleteKey(String key);
 
   /// Get the MeiliSearch version
   Future<Map<String, String>> getVersion();
 
   /// Get all index stats.
   Future<AllStats> getStats();
+
+  /// Get all tasks.
```

Comment:
```suggestion
  /// Get a list of tasks from the client.
```
For the moment all tasks are retrieved but soon the list will be paginated.